tests/unit/dialog/helper.js

```diff
@@ -49,9 +46,11 @@ return $.extend( helper, {
 		widthAfter = element.width();
 
 		msg = msg ? msg + "." : "";
-		actual = { width: widthAfter, height: heightAfter },
-		expected = { width: widthBefore + dw, height: heightBefore + dh };
-		assert.deepEqual( actual, expected, "resized[" + 50 + ", " + 50 + "] " + msg );
+
+		actualDH = heightAfter - heightBefore;
+		actualDW = widthAfter - widthBefore;
+
+		assert.ok( Math.abs(actualDH - dh) <= 1 && Math.abs(actualDW - dw) <= 1, "resized[50, 50] " + msg );
```

Comment:
I see. Let's add a TODO comment then, for whenever we end up having to adjust all the tests anyway.